cpp/velox/shuffle/VeloxShuffleWriter.cc

```diff
@@ -1611,4 +1666,20 @@ arrow::Status VeloxShuffleWriter::splitFixedWidthValueBuffer(const velox::RowVec
     return shrunken;
   }
 
+  void VeloxShuffleWriter::stat() const {
+#if VELOX_SHUFFLE_WRITER_LOG_FLAG
```

Comment:
Do we want to report those metrics like [others](https://github.com/oap-project/gluten/blob/da216c134470579f8e7019ab3f69802d94ea0370/cpp/core/shuffle/LocalPartitionWriter.cc#L216)?

Reply:
I want to get more information about shuffle writing for tracing performance issues now, You are welcome to report this information as metrics through JNI calling like  doing in `Java_io_glutenproject_vectorized_ShuffleWriterJniWrapper_stop`